packages/asset-service/src/generateAssetData/ethereum/idleVaults.ts

```diff
@@ -0,0 +1,70 @@
+import { ethChainId as chainId, toAssetId } from '@shapeshiftoss/caip'
+import { IdleSdk, IdleVault } from '@shapeshiftoss/investor-idle'
+import toLower from 'lodash/toLower'
+
+import { Asset } from '../../service/AssetService'
+import { ethereum } from '../baseAssets'
+import { colorMap } from '../colorMap'
+
+// const network = 1 // 1 for mainnet
+// const provider = new JsonRpcProvider(process.env.REACT_APP_ETHEREUM_NODE_URL)
```

Comment:
@samster91 will we need these?

Reply:
i think this is pasta from another impl

Reply:
Yeah but looks like the idle package won't need these to work, it doesn't accept these in its constructor like the yearn SDK would. Let's remove them on the next PR